package.json

```diff
@@ -20,9 +20,11 @@
     "setup"
   ],
   "dependencies": {
+    "@iobroker/adapter-core": "^1.0.3",
     "body-parser": "^1.19.0",
     "connect-flash": "^0.1.1",
     "cookie-parser": "^1.4.4",
+    "dock-spawn-ts": "^2.18.0",
```

Comment:
I think this must be in devDependencies.

Reply:
why???
it will be needed on runtime.
or do you mean cause it's copied during build

Reply:
Yes